src/java2yaml/DataContracts/ConfigModel/JavadocOptionModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace Microsoft.Content.Build.Java2Yaml
+{
+    using System.Text;
+
+    public class JavadocOptionModel
+    {
+        public string ClassPath { get; set; }
+        public string Encoding => " UTF-8";
+        public string DocletPath { get; set; }
+        public string Doclet => Constants.DocletClass;
+        public string SourcePath { get; set; }
+        public string OutputPath { get; set; }
+
+        public JavadocOptionModel() {}
+
+        public static JavadocOptionModel Create(
+            JavadocOptionModel option,
+            string dependencies,
+            string docletPath,
+            string sourcePath,
+            string outputPath)
+        {
+            option.ClassPath = dependencies;
+            option.DocletPath = docletPath;
+            option.SourcePath = sourcePath;
+            option.OutputPath = sourcePath;
+
+            return option;
+        }
+
+        public static string ToCmdArguments(JavadocOptionModel option)
+        {
+            var optionBuilder = new StringBuilder();
+
+            foreach (var prop in typeof(JavadocOptionModel).GetProperties())

```

Comment:
We can enumerate properties here. Usually we try avoiding reflection like `GetProperties`, as it is slow.

Reply:
change to build argument string in explicit way 
``` 
``` 
return "  -classpath " + dependencies
	+ " -encoding UTF-8"
	+ " -docletpath " + docletPath
	+ " -doclet com.microsoft.doclet.DocFxDoclet"
	+ " -sourcepath " + sourcePath
	+ " -outputpath " + outputPath;
```